i18n/fr/docusaurus-plugin-content-docs/current/wallet/Desktop-Wallet-Guide.md

```diff
@@ -3,10 +3,6 @@ sidebar_position: 30
 title: Desktop Wallet Guide
 ---
 
-:::info
-Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page.
-:::
-
```

Comment:
Why did you remove this?

Reply:
I did it inadvertently. Good catch, thanks :)

Reply:
fixed!